migration-rules/v0.9.28/tests/src/test/scala/fix/RuleSuite.scala

```diff
@@ -0,0 +1,8 @@
+package fix
+
+import scalafix.testkit.AbstractSemanticRuleSuite
+import org.scalatest.FunSuiteLike
+
+class wRuleSuite extends AbstractSemanticRuleSuite with FunSuiteLike {
```

Comment:
not that it matters, but it caught my attention
```suggestion
class RuleSuite extends AbstractSemanticRuleSuite with FunSuiteLike {
```